amazon-kinesis-connector-flink/src/main/java/software/amazon/kinesis/connectors/flink/metrics/ShardConsumerMetricsReporter.java

```diff
@@ -75,4 +79,11 @@ public void setNumberOfDeaggregatedRecords(int numberOfDeaggregatedRecords) {
 		this.numberOfDeaggregatedRecords = numberOfDeaggregatedRecords;
 	}
 
+	public void unregister() {
+		((GenericMetricGroup) this.metricGroup).close();
+	}
+
+	public boolean isRegistered() {
+		return !((GenericMetricGroup) this.metricGroup).isClosed();
+	}
```

Comment:
This method is not used by non test code. Can we delete it?

Reply:
Good catch, I'm removing it.

Reply:
Good catch, I'm removing it.